hockey/hockey2.php

```diff
@@ -0,0 +1,79 @@
+<?php
+include('config/init.php');
+
+echo "
+<head>
+    <link rel='stylesheet' href='/hockey/hockey.css'>
+    </head>
+    <body>
+    <h1><div class='stats'>NHL Skater Stats 2016-2017</div>
+    <div class='reset'><a href='/hockey/hockey2.php'>reset</a></div></h1>
+        <table>
+            <tr class='tableHeader'>
+                <th> <span class='word'>Rank</th>";
+
+tableHeader('player','Player');
+tableHeader('pos','Position');
+tableHeader('team','Team City');
+tableHeader('gp','Games Played');
+tableHeader('toi','Time on Ice');
+tableHeader('gf','Goals For');
+tableHeader('ga','Goals Against');
+tableHeader('sf','Shots For');
+tableHeader('sa','Shots Against');
+echo '</tr>';
+
+$display = '';
+
+if(isset($_REQUEST['page'])){
+    $display .= displayResults();
+}
+elseif(isset($_REQUEST['sort'])){
+    $display .= displayResultsSort('player','LastName','ASC');
+    $display .= displayResultsSort('pos','Pos','ASC');
+    $display .= displayResultsSort('team','Team','ASC');
+    $display .= displayResultsSort('gp','GP','DESC');
+    $display .= displayResultsSort('toi','TOI','DESC');
+    $display .= displayResultsSort('gf','GF','DESC');
+    $display .= displayResultsSort('ga','GA','DESC');
+    $display .= displayResultsSort('sf','SF','DESC');
+    $display .= displayResultsSort('sa','SA','DESC');
+
+}
+
+elseif(isset($_REQUEST['filter'])){
+    if (empty($_REQUEST['Pos']) && empty($_REQUEST['Team']) && empty($_REQUEST['GP'])){
+        echo tableContents(0,2);
+    }
+    $filter =array(
```

Comment:
We should talk through how this `$filter` approach is set up. I assume it works which is a great start (this is pretty complicated code) but there might be a more efficient way to do it. When you have some time, can you set up a 1:1 on the calendar? Make sure you're comfortable walking me through the approach you took here before the meeting and I'll propose some ways to refine it.